classes/user/DAO.php

```diff
@@ -80,6 +80,8 @@ class DAO extends EntityDAO
 
     /**
      * Construct a new User object.
+     *
+     * @return User
```

Comment:
Since I think this PR is most suitable for `main`, I'd rather we just started adding first-class type hints.